CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 * [#2408](https://github.com/ruby-grape/grape/pull/2408): Fix params method redefined warnings - [@ericproulx](https://github.com/ericproulx).
 * [#2410](https://github.com/ruby-grape/grape/pull/2410): Gem deprecations will raise a DeprecationWarning in specs - [@ericproulx](https://github.com/ericproulx).
 * [#2389](https://github.com/ruby-grape/grape/pull/2389): Remove rack-accept dependency - [@ericproulx](https://github.com/ericproulx).
+* [#2425](https://github.com/ruby-grape/grape/pull/2425): Replace `{}` with `Rack::Header` (for rack 3.0 and greater) or Rack::Utils::HeaderHash` (for rack less than 3.0) in Grape::DSL::Headers - [@dhruvCW](https://github.com/dhruvCW).
```

Comment:
Quote these consistently, `Grape::DSL::Headers`, maybe shorten.

Replace `{}` with `Rack::Header` or `Rack::Utils::HeaderHash`.

or if you really want versions here

Replace `{}` with `Rack::Header` in Rack 3.0+ and Rack::Utils::HeaderHash` for Rack 1.x and 2.x.

Reply:
yup the one without version sounds better 😅 replaced.